app/src/main/java/com/devshish/internship/domain/repository/ISongsRepository.kt

```diff
@@ -4,5 +4,7 @@ import com.devshish.internship.domain.model.Song
 
 interface ISongsRepository {
 
+    var songToPlay: Song?
```

Comment:
It is not good to create such val inside of Songs Repository
1. Because our repository provide us information about songs, but do not provide us state about currently played song 
2. What song is playing should be holded somewhere else